### ISSUE TITLE
fix package-systemd

### ISSUE THREAD

**Issue by pnpavlov** [gardenlinux/gardenlinux#2497]
https://github.com/gardenlinux/package-systemd/actions/runs/120213
```
patching file debian/systemd.install
Hunk #1 FAILED at 4.
1 out of 1 hunk FAILED -- saving rejects to file debian/systemd.install.rej
```

**Comment by 5kt** [comment]
Duplicate of : https://github.com/gardenlinux/gardenlinux/issues/2491

**Comment by fwilhe** [comment]
I've marked the PR as a draft. I guess it will be useful to start from there once the non-rc release is ready. Moving this issue to blocked until debian has a version in testing that we want to use again.
